Frameworks/UIKit/UIDatePicker.mm

```diff
@@ -814,6 +813,8 @@ static void resetPickerPositions(UIDatePicker* self) {
  @Status Interoperable
 */
 - (void)setMinimumDate:(NSDate*)date {
+    [date retain];
+    [_minimumDate release];
     _minimumDate = date;
```

Comment:
Actually I think if we just define our own getter, the compiler will turn off synthesis. That is the problem. The getter isn't defined so the compiler has to synthesize one with an ivar (one underscore and raw*). If we instead define it ourselves and return the strong id, no extra ivar will be produced, but we will have to change internal uses of self->_min/maxdate as those won't exist now. Basically we either need to go full custom with StrongId or full synthesized and I'm arguing for Strong id since it cleans up the memory management and we don't get the atomicity benefits of synthesized since they are declared as nonatomic anyway. How does that sound?

---
In reply to: [74986350](https://github.com/Microsoft/WinObjC/pull/883#discussion_r74986350) [](ancestors = 74986350,74983076)